<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #333333;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: rgba(51, 51, 51, 150);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="LOGO">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(223, 223, 223);
	border: none;
	border-radius: 5px;
}
QPushButton:hover{
	color: white;
	background-color: rgb(203, 203, 203);
}
QPushButton:pressed{
	background-color: #333333;
	border: 2px solid rgb(203, 203, 203);
}</string>
    </property>
    <property name="text">
     <string>Take Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(223, 223, 223);
	border: none;
	border-radius: 5px;
}
QPushButton:hover{
	color: white;
	background-color: rgb(203, 203, 203);
}
QPushButton:pressed{
	background-color: #333333;
	border: 2px solid rgb(203, 203, 203);
}</string>
    </property>
    <property name="text">
     <string>Record Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(223, 223, 223);
	border: none;
	border-radius: 5px;
}
QPushButton:hover{
	color: white;
	background-color: rgb(203, 203, 203);
}
QPushButton:pressed{
	background-color: #333333;
	border: 2px solid rgb(203, 203, 203);
}</string>
    </property>
    <property name="text">
     <string>Open Active Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(223, 223, 223);
	border: none;
	border-radius: 5px;
}
QPushButton:hover{
	color: white;
	background-color: rgb(203, 203, 203);
}
QPushButton:pressed{
	background-color: #333333;
	border: 2px solid rgb(203, 203, 203);
}</string>
    </property>
    <property name="text">
     <string>Exit Program</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(223, 223, 223);</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sub">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="zoom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="add">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: white;
	background-color: rgba(51, 51, 51, 150);
}</string>
   </property>
   <property name="text">
    <string>User Name:&lt;br&gt;Patient ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: #fffbbd;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>video</zorder>
  <zorder>frame</zorder>
  <zorder>id</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
